packages/espressocash_api/lib/src/dto/withdraw.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:espressocash_api/espressocash_api.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'withdraw.freezed.dart';
+part 'withdraw.g.dart';
+
+@freezed
+class WithdrawPaymentRequestDto with _$WithdrawPaymentRequestDto {
+  const factory WithdrawPaymentRequestDto({
+    required String senderAccount,
```

Comment:
For new endpoints, `senderAccount` is not actually needed, API requests are signed by the wallet anyway.